script.js

```diff
@@ -1,4 +1,118 @@
+var computerHand = function () {
+  // output a random integer from 1 to 3 such that each integer represent a different Hand
+  var randomNumber = Math.random() * 3;
+  var randomInteger = Math.ceil(randomNumber);
+  if (randomInteger == 1) {
+    return `scissors`;
+  }
+  if (randomInteger == 2) {
+    return `stone`;
+  }
+  if (randomInteger == 3) {
+    return `paper`;
+  }
+};
+var whoWins = function (input, computerUse) {
+  console.log(`computer hand = ${computerUse}`);
+  // this is if the computer wins
+  if ((computerUse == `scissors` && input == `paper`) || (computerUse == `paper` && input == `stone`) || (computerUse == `stone` && input == `scissors`)) {
+    playcount = playcount + 1;
+    computerWin = computerWin + 1;
+    possibleChampion = " the computer ";
+    console.log(`you lose`);
+    var outcome = `you lose!`;
+  }
+
+  // this is if the human wins
+  else if ((input == `scissors` && computerUse == `paper`) || (input == `paper` && computerUse == `stone`) || (input == `stone` && computerUse == `scissors`)) {
+    humanWin = humanWin + 1;
+    playcount = playcount + 1;
+    possibleChampion = " you";
+    console.log(`you win`);
+    var outcome = `you win!`;
+  }
+
+  //if there is a draw, the first duel will have no ultimate winner
+  else if (input == computerUse) {
+    if (playcount == 0) {
+      console.log(`draw`);
+      possibleChampion = " no one";
+      var outcome = `you drew with the computer`;
+    }
```

Comment:
If this was the first condition. you can leave all the losing conditions to the else at the back. can write less code
good use of the || 

if (draw)
else if ( computer wins)
else //(user wins)